content/news/030/index.md

```diff
@@ -206,6 +206,41 @@ for expressively abstracting over user input.
 Supports local multiplayer, enables input rebinding, integrates with `bevy_ui`,
 and handles chords!
 
+### [Edict]
+
+New archetypes based ECS implementation by [@zakarumych](https://github.com/zakarumych).
+
+The novel feature of [Edict] is entity ownership implemented via reference counting.\
+This optional feature allows creating owned kind of `Entity` "reference",
+that ensures entity is alive and despawns it on drop.\
+Storing owning `Entity` in the component of another entity
+creates ownership relation between those entities.\
+Even though `Entity` is an owning reference,
+components of the entity are can be queried from `World` as usual.\
+Shared ownership is also available.
+
+Optimized for both high density `World`s with thousands of entities
+and also for `World`s with lower number of entities spread among many archetypes.\
+[Edict] is aimed for wide range of game genres and use cases outside of games.
+
+Built-in change detection with epochs allows systems to query for components
+that were updated since last run of that query,
+or since any other epoch as defined by `Tracks` argument.\
```

Comment:
are these explicit line breaks really needed here?

Reply:
No, I just thought it makes text easier to read.
I'm not an expert writer, so feel free to change formatting the way you think is better.

Reply:
@17cupsofcoffee wdyt? i think it'd be better and more inline with our guidelines ("Do not use rich formatting (bold, italics, etc).") to keep the formatting as simple as possible.

Reply:
@ozkriff  :+1: I think keep it simple - the website renders the text in a bigger font and with word wrap/a narrower column width, so I think it will look fine without the line breaks in context.